deployment/eks/1.initialize-database.sql

```diff
@@ -0,0 +1,44 @@
+USE eventuate;
```

Comment:
These SQL files are duplicates of files https://github.com/eventuate-foundation/eventuate-common/tree/master/mssql - please access via versioned URL

Reply:
Done.